Resources/Textures/Structures/Storage/wall_locker.rsi/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "https://github.com/space-wizards/space-station-14/pull/10673",
```

Comment:
Did you make this?

Reply:
Yes I did, but it's based off of one of the locker door textures

Reply:
Add in a "modified from XYZ" then to attribute the source.